qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/prefect.tf

```diff
@@ -0,0 +1,19 @@
++resource "helm_release" "prefect" {
++  name      = "prefect"
++  namespace = var.environment
++  chart     = "../charts/prefect"
```

Comment:
This does not exists here, we need to create a prefect module in [terraform modules](https://github.com/Quansight/qhub-terraform-modules) and call that module.

Reply:
Also, would need to add some usage documentation.

Reply:
https://github.com/Quansight/qhub-datum-gcp-prod_old_2020-11-12/blob/master/infrastructure/prefect.tf

Reply:
Example in infrastructure/prefect.tf for old client deployment